webapp/src/main/webapp/vue-apps/space-form/components/SpaceFormDrawer.vue

```diff
@@ -402,23 +402,27 @@ export default {
       this.open();
     },
     openByEvent(e) {
-      this.openByRootEvent(e?.detail);
+      this.openByRootEvent(e?.detail?.templateId, e?.detail?.spaceTemplates);
     },
-    openByRootEvent(templateId) {
+    openByRootEvent(templateId, spaceTemplates) {
       this.goBackButton = !templateId;
-      this.open(templateId);
+      this.open(templateId, spaceTemplates);
     },
-    async open(templateId) {
+    async open(templateId, spaceTemplates) {
       this.templateId = templateId && Number(templateId);
       this.space = {
         templateId: templateId,
         subscription: 'open',
         visibility: 'private',
       };
+      if (spaceTemplates) {
+        this.templates = spaceTemplates;
+      } else {
+        this.templates = this.$root.spaceTemplates;
```

Comment:
It's important to keep it as it was:

```suggestion
        if (!this.$root.spaceTemplates) {
          this.$root.spaceTemplates = await this.$spaceTemplateService.getSpaceTemplates();
        }
        this.templates = this.$root.spaceTemplates;
```